x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/integrations/agent_required_callout.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useCallback } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiIcon } from '@elastic/eui';
+
+import { LinkAnchor } from '../../../../../common/components/links';
+import { CardCallOut } from '../common/card_callout';
+import { useNavigation } from '../../../../../common/lib/kibana';
+import { FLEET_APP_ID, ADD_AGENT_PATH } from './const';
+
+export const AgentRequiredCallout = React.memo(() => {
+  const { getAppUrl, navigateTo } = useNavigation();
+  const addAgentLink = getAppUrl({ appId: FLEET_APP_ID, path: ADD_AGENT_PATH });
+  const onAddAgentClick = useCallback(() => {
+    navigateTo({ appId: FLEET_APP_ID, path: ADD_AGENT_PATH });
```

Comment:
NIT: we could extract:
`const fleetAgentLinkProps = { appId: FLEET_APP_ID, path: ADD_AGENT_PATH };`
outside the component